array-agg/docker-compose.yaml

```diff
@@ -0,0 +1,84 @@
+version: '2.1'
+services:
+  mysql:
+    image: debezium/example-mysql:2.6
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=123456
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysqlpw
+      - TZ=UTC # America/Los_Angeles
+    volumes:
+     - ./mysql/ecom.sql:/docker-entrypoint-initdb.d/ecom.sql
+    networks:
+      - my-network
+  opensearch-node1: # This is also the hostname of the container within the Docker network (i.e. https://opensearch-node1/)
+    image: opensearchproject/opensearch:1 # Specifying the latest 1.x available image - modify if you want a specific version
+    container_name: opensearch-node1
+    environment:
+      - cluster.name=opensearch-cluster # Name the cluster
+      - node.name=opensearch-node1 # Name the node that will run in this container
+      - discovery.seed_hosts=opensearch-node1,opensearch-node2 # Nodes to look for when discovering the cluster
+      - cluster.initial_master_nodes=opensearch-node1,opensearch-node2 # Nodes eligible to serve as master
+      - bootstrap.memory_lock=true # Disable JVM heap memory swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # Set min and max JVM heap sizes to at least 50% of system RAM
+      #- "DISABLE_INSTALL_DEMO_CONFIG=true" # Prevents execution of bundled demo script which installs demo certificates and security configurations to OpenSearch
+      #- "DISABLE_SECURITY_PLUGIN=true" # Disables security plugin
```

Comment:
With the security plugin enabled, since the SSL certificate is unsigned, Decodable won't be able to connect to OpenSearch unless it's proxied via ngrok (as used in the article) or something similar that will accept an unsigned certificate:

> Invalid connection. Reason: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

We can try the following instead:
```yaml
      - plugins.security.ssl.http.enabled=false
```
It will disable SSL but will keep user authentication in place. In my experience, if this plugin is fully disabled, then OpenSearch Dashboards cannot authenticate to Opensearch ([problem](https://forum.opensearch.org/t/initial-setup-security-plugin-error/15366/15), [solution](https://forum.opensearch.org/t/initial-setup-security-plugin-error/15366/15)).